astronomer/providers/snowflake/sensors/snowflake.py

```diff
@@ -0,0 +1,101 @@
+from __future__ import annotations
+
+from datetime import timedelta
+from typing import Any, Callable, Dict, List, Optional, Sequence, Union
+
+from airflow import AirflowException
+from airflow.sensors.base import BaseSensorOperator
+
+from astronomer.providers.snowflake.triggers.snowflake_trigger import (
+    SnowflakeSensorTrigger,
+)
+from astronomer.providers.utils.typing_compat import Context
+
+
+class SnowflakeSensorAsync(BaseSensorOperator):
+    """
+    Runs a sql statement repeatedly until a criteria is met. It will keep trying until
+    success or failure criteria are met, or if the first cell is not in (0, '0', '', None).
```

Comment:
Can you please elaborate on this `if the first cell is not in (0, '0', '', None)` - not sure whether user would understand it